spec/requests/api/v1/users_spec.rb

```diff
@@ -10,7 +10,14 @@
     let(:parsed_response) { JSON.parse(response.body) }
 
     before :each do
-      3.times { create(:user) }
+      # Heads up, for some reason the stored CIP IDs don't match
```

Comment:
Why would they not match, argh!

If you do `cip = CoreInductionProgramme.create!(name: "Teach First")` do they match? (I hope it's some funny behaviour by factory bot :/ )

Reply:
Dunno but it doesn't affect the tests or the current output of the API so I figured I'd just leave a note - don't want to spend time investigating it if it never affects anyone!